From ec5164b8a8e282b91aedb2af94dfeb89887656b7 Mon Sep 17 00:00:00 2001
From: turboderp <11859846+turboderp@users.noreply.github.com>
Date: Tue, 19 Sep 2023 11:56:52 +0200
Subject: [PATCH] Fix for non-numerical version number in flash-attn

---
 exllamav2/attn.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/exllamav2/attn.py b/exllamav2/attn.py
index b8b1752..09ef6d6 100644
--- a/exllamav2/attn.py
+++ b/exllamav2/attn.py
@@ -16,7 +16,7 @@
 has_flash_attn = False
 try:
     import flash_attn
-    flash_attn_ver = [int(t) for t in flash_attn.__version__.split(".")]
+    flash_attn_ver = [int(t) for t in flash_attn.__version__.split(".") if t.isdigit()]
     if flash_attn_ver >= [2, 2, 1]:
         from flash_attn import flash_attn_func
         has_flash_attn = True

